src/micro_http/src/common/headers.rs

```diff
@@ -286,6 +312,174 @@ impl Headers {
     }
 }
 
+/// Wrapper over supported AcceptEncoding.
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub enum Encoding {
```

Comment:
We only need the `Identity` variant of this enum as it is the only one we currently support. In this case we can consider removing the variants altogether and only keeping the methods.